manifest.webapp

```diff
@@ -1,7 +1,7 @@
 {
   "name": "j2me.js",
   "description": "j2me interpreter for firefox os",
-  "launch_path": "/index.html?autosize=1&logConsole=web",
+  "launch_path": "/index.html",
```

Comment:
It looks like we don't default to autosize=1 anymore; does this cause apps to appear incorrectly on device and/or in emulators?